assets/dev/js/frontend/handlers/video.js

```diff
@@ -27,6 +27,15 @@ class VideoModule extends elementorModules.frontend.handlers.Base {
 		if ( ! this.getElementSettings( 'lightbox' ) ) {
```

Comment:
You should handle all cases, including lightbox

Reply:
But the Lightbox module handles its own videos